index.html

```diff
@@ -201,8 +201,27 @@ <h3>Skills</h3>
           </ul>
         </section>
       </section>
+
+      <footer class="contact-wrapper">
+        <h2 class="contact-title">Contact me</h2>
```

Comment:
- [ ] Please kindly correct this font style, since it's missing font size, weight, etc.  In order to get the values, get them on the Figma (right side of the template after selecting an element).  That way, your project will look more professional 💪